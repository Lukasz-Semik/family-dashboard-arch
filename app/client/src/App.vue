<template>
  <div id="app">
    <p :class="$style['test']">App</p>
  </div>
</template>

<style lang="scss" module src="./App.scss" />