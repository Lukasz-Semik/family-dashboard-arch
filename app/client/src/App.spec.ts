import { shallowMount } from '@vue/test-utils';

import App from './App.vue';

describe('<App />', () => {
  const wrapper = shallowMount(App);
  it('should rener properly', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
