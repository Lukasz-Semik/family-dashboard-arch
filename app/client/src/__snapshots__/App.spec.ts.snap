// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<App /> should rener properly 1`] = `
<div id="app">
  <p class="test">App</p>
</div>
`;
